modelforge/train/training.py

```diff
@@ -942,13 +914,22 @@ def perform_training(
     import toml
 
     dataset_statistic = toml.load(dm.dataset_statistic_filename)
-    log.info(dataset_statistic["training_dataset_statistics"])
+    log.info(
+        f"Setting per_atom_energy_mean and per_atom_energy_stddev for {potential_config.potential_name}"
+    )
+    log.info(
+        f"per_atom_energy_mean: {dataset_statistic['training_dataset_statistics']['per_atom_energy_mean']}"
+    )
+    log.info(
+        f"per_atom_energy_stddev: {dataset_statistic['training_dataset_statistics']['per_atom_energy_stddev']}"
+    )
+
     # Set up model
     model = NeuralNetworkPotentialFactory.generate_model(
         use="training",
         dataset_statistic=dataset_statistic,
-        model_parameter=potential_config,
-        training_parameter=training_config["training_parameter"],
+        model_parameter=potential_config.model_dump(),
+        training_parameter=training_config.model_dump(),
```

Comment:
I have a slight preference for `dict(potential_config)` instead of `.model_dum()` since it is more verbose of what is happening

Reply:
I mentioned this in a comment above. model_dump() is required for a nested model.  As we just discussed in slack, I think after this release, we should work to moving towards an approach where all main entry points accept the pydantic models, rather than variables with defaults.  This will provide a much better means of validating inputs and tracking.